Source/ToolCore/JSBind/JSBTypeScript.cpp

```diff
@@ -349,28 +349,8 @@ void JSBTypeScript::ExportModuleEnums(JSBModule* module)
         while (valsIter != values.End())
         {
             String name = (*valsIter).first_;
-            String value = (*valsIter).second_;
-
-            // BodyType2D enum order is assigned in RigidBody2D.h in incorrect order
-            if (_enum->GetName() == "BodyType2D")
-            {
-                if (name == "BT_STATIC")
-                    value = "0";
-                else if (name == "BT_KINEMATIC")
-                    value = "1";
-                else if (name == "BT_DYNAMIC")
-                    value = "2";
-            }
-
```

Comment:
Nice cleanup :)  This is something that would be nice to be able to do in C#, though we can't due to constant requirements